ros2cli/ros2cli/xmlrpc/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import xmlrpc.client as client
```

Comment:
Why should be expose `client` and mask it as our own?

Reply:
For consistency, and to force client code to indirectly import all our custom `xmlrpc` (un)marshalling code.

Reply:
Please add a comment for this rational.

Reply:
Done in 7750f7c3798d2c427f7ef434a366b631787951f2.